src/main/java/com/cleanroommc/bogosorter/common/sort/DefaultRules.java

```diff
@@ -27,8 +28,9 @@ public static void init(IBogoSortAPI api) {
         api.registerNbtSortingRule("potion", "Potion", Constants.NBT.TAG_STRING, ItemCompareHelper::comparePotionId, DefaultRules::getPotionId);
         api.registerNbtSortingRule("enchantment", "ench", Constants.NBT.TAG_LIST, ItemCompareHelper::compareEnchantments, nbtBase -> (NBTTagList) nbtBase);
         api.registerNbtSortingRule("enchantment_book", "StoredEnchantments", Constants.NBT.TAG_LIST, ItemCompareHelper::compareEnchantments, nbtBase -> (NBTTagList) nbtBase);
-        if (Loader.isModLoaded("gregtech")) {
+        if ( BogoSorter.isGTCEuLoaded() ) {
```

Comment:
I dont see a reason to change this. This allows supporting gtce and gtceu at the same time without having to depend on either

Reply:
That can work